### DISCUSSION TITLE
installing prev versions

### DISCUSSION THREAD

**Discussion by pouyaSanjari** [FranzKafkaYu/x-ui#371] [Q&A] [answered]
hello
i want to install prev versions
is it possible and how?

**Comment by pouyaSanjari** [comment]
> hello i want to install prev versions is it possible and how?

actually i want downgrade

**Reply by FranzKafkaYu** [reply, marked as answer]
> > hello i want to install prev versions is it possible and how?
> 
> actually i want downgrade

bash <(curl -Ls https://raw.githubusercontent.com/FranzKafkaYu/x-ui/master/install.sh)   VERSION_NAME  
For example    
```  
bash <(curl -Ls https://raw.githubusercontent.com/FranzKafkaYu/x-ui/master/install.sh)  0.3.4.1
```
